package.json

```diff
@@ -83,6 +83,7 @@
     "gulp-rename": "^2.0.0",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-uglify": "^3.0.2",
+    "gulp-webp": "^4.0.1",
```

Comment:
Last update was 01/23/19. We should keep a close eye on this in case dependabot alerts start popping up.